README.md

```diff
@@ -1,113 +1,138 @@
 <div id="top"></div>
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/rotki/rotki/develop/frontend/app/public/assets/images/rotkehlchen_no_text.png" alt="An open source portfolio tracker, accounting and analytics tool that protects your privacy" width="250">
+  <img src="https://raw.githubusercontent.com/rotki/rotki/develop/frontend/app/public/assets/images/rotkehlchen_no_text.png" alt="Rotki Logo" width="250">
 </p>
 
+<h1 align="center">Rotki: The Open-Source Portfolio Manager</h1>
+
+<p align="center">
+  <strong>A privacy-focused portfolio tracker, accounting, and analytics tool.</strong>
+</p>
 
 <div align="center">
 
 [![GitHub release](https://img.shields.io/github/release/rotki/rotki.svg)](https://GitHub.com/rotki/rotki/releases/)
-[![Docker Image Version (tag latest semver)](https://img.shields.io/docker/v/rotki/rotki/latest?label=Docker)](https://hub.docker.com/layers/rotki/rotki/dev/images/sha256-acbd52985ccea0fb42820a655d994312d322a0895ee5777733582b017a89f3b0?context=explore)
```

Comment:
This change looks weird. @kelsos can you check if the previous link was wrong?

Reply:
Interesting how did this end up pointing to a layer of the dev image?